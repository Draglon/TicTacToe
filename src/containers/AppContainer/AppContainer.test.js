import React from 'react';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';

import ConnectedApp, { AppContainer } from './AppContainer';

describe('<AppContainer /> container', () => {
  const mockStore = configureStore();
  const initialState = {};
  let store;
  let container;

  beforeEach(() => {
    store = mockStore(initialState);
    container = shallow(<ConnectedApp store={store} />);
  });

  it('render the connected(SMART) component', () => {
    expect(container.length).toEqual(1);
  });

  it('render property', () => {
    expect(container).toMatchSnapshot();
  });
});
