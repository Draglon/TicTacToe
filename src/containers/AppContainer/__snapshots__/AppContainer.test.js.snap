// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppContainer /> container render property 1`] = `ShallowWrapper {}`;
